tests/integration/test_write_forecasts.py

```diff
@@ -145,8 +144,8 @@ def test_write_and_read_forecast(self):
         # Assert
         result = self.database.get_predicted_load(
             pj={"id": 308, "resolution_minutes": 15},
-            start_time=datetime(2022, 1, 1, 1, 0, 0),
-            end_time=datetime(2022, 1, 1, 2, 0, 0),
+            start_time=datetime(2022, 1, 1, 1, 0, 0) - timedelta(days=1),
```

Comment:
Why do you subtract a day from `start_time` and add a day to `end_time`?
Wouldn't it be a better test if the `start_time` and `end_time` arguments exactly match the boundaries of the interval of timeseries data that we want to retrieve from influx?